src/main/scala/com/wavesplatform/database/Keys.scala

```diff
@@ -154,4 +156,6 @@ object Keys {
       writeTransactionHN
     )
 
+  val DisableTxsByAddress =
```

Comment:
This option should be configurable from `waves.conf`. There's no need to store this value in LevelDB.

Reply:
It should be persisted to force user to rebuild the state, if he re-enables transactions by address.